typed-store/src/rocks/tests.rs

```diff
@@ -62,6 +62,88 @@ fn test_get() {
     assert_eq!(None, db.get(&000000000).expect("Failed to get"));
 }
 
+#[test]
+fn test_multi_get() {
+    let db = DBMap::open(temp_dir(), None, None).expect("Failed to open storage");
+
+    db.insert(&123, &"123".to_string())
+        .expect("Failed to insert");
+    db.insert(&456, &"456".to_string())
+        .expect("Failed to insert");
+
+    let result = db.multi_get(&[123, 456, 789]).expect("Failed to multi get");
+
+    assert_eq!(result.len(), 3);
+    assert_eq!(result[0], Some("123".to_string()));
+    assert_eq!(result[1], Some("456".to_string()));
+    assert_eq!(result[2], None);
+}
+
+#[test]
+fn test_seek() {
+    let db = DBMap::open(temp_dir(), None, None).expect("Failed to open storage");
+
+    db.insert(&123, &"123".to_string())
+        .expect("Failed to insert");
+    db.insert(&456, &"456".to_string())
+        .expect("Failed to insert");
+    db.insert(&789, &"789".to_string())
+        .expect("Failed to insert");
+
+    // Skip all smaller
+    let key_vals: Vec<_> = db.iter().skip_to(&456).expect("Seek failed").collect();
+    assert_eq!(key_vals.len(), 2);
+    assert_eq!(key_vals[0], (456, "456".to_string()));
+    assert_eq!(key_vals[1], (789, "789".to_string()));
+
+    // Skip to the end
+    let key_vals: Vec<_> = db.iter().skip_to(&999).expect("Seek failed").collect();
+    assert_eq!(key_vals.len(), 0);
+
+    // Skip to the one before the end
+    let key_vals: Vec<_> = db
+        .iter()
+        .skip_prior_to(&999)
+        .expect("Seek failed")
+        .collect();
+    assert_eq!(key_vals.len(), 1);
+    assert_eq!(key_vals[0], (789, "789".to_string()));
+
+    // Skip to pror of first value
+    let key_vals: Vec<_> = db.iter().skip_to(&000).expect("Seek failed").collect();
+    assert_eq!(key_vals.len(), 3);
+
+    // The counter intuitive ones
+
+    // Skip to pror of first value
+    let key_vals: Vec<_> = db
+        .iter()
+        .skip_prior_to(&000)
+        .expect("Seek failed")
+        .collect();
+    assert_eq!(key_vals.len(), 0);
+}
```

Comment:
Nit: I would move the following calls to the next `#[test]`:
```
.skip_prior_to(&999)
.skip_prior_to(&000)
```

I would keep the following, but rename its comment, which does not match the call:
```
.skip_to(&000)
```